dmarc.tex

```diff
@@ -0,0 +1,81 @@
+\section{Data Management Conceptual Architecture \label{sect:dmarc}}
+
+The DM Subsystem Architecture is detailed in \citeds{LDM-148}.
+A few of the higher level diagrams are reproduced here to orientate the reader within DM.
+
+During Operations, components of the DM Subsystem will be installed and run in
+multiple locations. These include:
+
+\begin{itemize}
+\item The Commissioning Cluster, which may be physically at NCSA in Urbana-Champaign
```

Comment:
I don't think there's any way that we will have the CC at NCSA.  It must be at the Base to talk to the OCS and EFD.